codespell_lib/data/dictionary.txt

```diff
@@ -34643,29 +34643,55 @@ refect->reflect
 refected->reflected
 refecting->reflecting
 refectiv->reflective
+refective->reflective
```

Comment:
2. Same for `refective`, I would move it to _rare_.
   * [Merriam-Webster](https://www.merriam-webster.com/dictionary/refective)